user/caribou/module/src/CLICTDEventConverter.cc

```diff
@@ -223,8 +223,8 @@ bool CLICTDEvent2StdEventConverter::Converting(eudaq::EventSPC d1, eudaq::Standa
       tot = 1;
     }
 
-    // Time defaults ot rising shutter edge:
-    auto timestamp = shutter_open;
+    // Time defaults to center between rising and falling shutter edge:
```

Comment:
I somehow remember fixing this already. Hm...

Reply:
I just realized, this whole thing is the same as MR !11